widgets/search-box/__tests__/search-box-test.js

```diff
@@ -182,7 +182,7 @@ describe('search-box()', () => {
         widget.getInput = sinon.stub().returns(input);
       });
 
-      it('performs the search on keyup', () => {
+      it('do not perform the search on keyup', () => {
```

Comment:
yes we should not perform the search on key up. I will add a test for input event